spec/queries/find_exact_query_spec.rb

```diff
@@ -290,8 +314,21 @@
     expect(query.exact_match).to be_nil
   end
 
+  it "matched on content_id only" do
+    query = build_query(content_id: "11a51f5e-3f17-4516-87c7-02afba87ef40")
+    subscriber_list = create_subscriber_list(content_id: "11a51f5e-3f17-4516-87c7-02afba87ef40")
+    expect(query.exact_match).to eq(subscriber_list)
+  end
+
+  it "not matched on different content_id" do
+    query = build_query(content_id: "11a51f5e-3f17-4516-87c7-02afba87ef40")
+    _subscriber_list = create_subscriber_list(content_id: "ef8a3fd2-2b15-4c0e-a2d4-2a34187f337a")
+    expect(query.exact_match).to be_nil
+  end
+
```

Comment:
Can you add a test to this file that a find_exact query with `content_id: ""` matches a subscriber_list with `content_id: nil`?

```ruby
  it "matched on blank content_id" do
    query = build_query(content_id: "")
    subscriber_list = create_subscriber_list(content_id: nil)
    expect(query.exact_match).to eq(subscriber_list)
  end
```

This currently fails.  I think in the initialiser we'll want

```ruby
@content_id = content_id.blank? ? nil : content_id
```

Reply:
Ah, `presence` works here:

```ruby
@content_id = content_id.presence
```